Added new font 8x10

jmamej opened the pull request:
Edited 7x10 font for better readability. Width of 7 pixels was left intentionally.

letter A (7x10): 0x1000, 0x2800, 0x2800, 0x2800, 0x2800, 0x7C00, 0x4400, 0x4400, 0x0000, 0x0000,
letter A (8x10): 0x3800, 0x4400, 0x4400, 0x4400, 0x7C00, 0x4400, 0x4400, 0x4400, 0x4400, 0x0000,

7x10:

............@................
........@....@............
........@....@............
........@....@............
........@....@............
....@@@@@........
....@............@........
....@............@........
................................
................................

8x10:

........@@@............
....@............@........
....@............@........
....@............@........
....@@@@@........
....@............@........
....@............@........
....@............@........
................................

jmamej committed:
Add new font 8x10

jmamej committed:
Add font to config

afiskon commented:
There seems to be a bug:

```
#ifdef SSD1306_INCLUDE_FONT_8x10
const SSD1306_Font_t Font_8x10 = {7, 10, Font8x10, NULL};
#endif
```

Also please add your font to ssd1306_tests.c

jmamej committed:
Update ssd1306_tests.h

Add compare function

jmamej committed:
Add font compare

Added function comparing 7x10 with 8x10.

jmamej commented:
Hi, i left width as 7 pixels intentionally, so it really is just 7x10 but more readable.

```
const SSD1306_Font_t Font_8x10 = {7, 10, Font8x10, NULL};
```

I added my font to ssd1306_TestFonts1, side by side with 7x10. (there was no space)
Also added ssd1306_TestCompareFonts that prints "Yx10 abcABC123$%&" for both fonts.
If You find 8x10 font more readable you can just replace contents of 7x10 without adding another font to the file.
